src/app/workflow/versions/versions.component.html

```diff
@@ -15,170 +15,164 @@
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            href="{{ Dockstore.DOCUMENTATION_URL }}/faq.html#what-is-a-default-version-of-a-tool-or-workflow"
+            ><span *ngIf="defaultVersion === version.name" class="label label-primary">Default</span></a
+          >
+          <i *ngIf="version.referenceType === 'BRANCH'" class="fa fa-code-fork" aria-hidden="true" title="Branch"></i>
+          <i *ngIf="version.referenceType === 'TAG'" class="fa fa-tag" aria-hidden="true" title="Tag"></i>
+          <a
+            *ngIf="_selectedVersion?.name !== version?.name; else alreadySelected"
+            matTooltip="Click to select version"
+            (click)="setVersion(version)"
+            >{{ version?.name }}</a
+          >
+          <a target="_blank" rel="noopener noreferrer" [href]="version.commitID | commitUrl: workflow?.providerUrl">
+            <mat-icon
+              *ngIf="version.commitID && (version.commitID | commitUrl: workflow?.providerUrl)"
+              [matTooltip]="version.commitID ? 'Commit ID: ' + version.commitID : ''"
+              >open_in_new</mat-icon
+            ></a
+          >
+        </span>
+        <ng-template #alreadySelected>
+          <span>{{ version?.name }}</span>
+        </ng-template>
+      </td>
+    </ng-container>
 
-            <span *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED">
-              <label placement="top" matTooltip="Version of the workflow.">
-                Version
-              </label>
-            </span>
-            <fa-icon class="pull-right" [icon]="getIconClass('reference')" (click)="clickSortColumn('reference')"></fa-icon>
-          </th>
+    <ng-container matColumnDef="last_modified">
+      <th scope="col" mat-header-cell *matHeaderCellDef mat-sort-header disableClear matTooltip="Date of last update to Git reference">
+        Date Modified
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <span matTooltip="{{ version.last_modified | date: 'medium' }}">{{ version.last_modified | date: 'mediumDate' }}</span>
+      </td>
+    </ng-container>
 
-          <th *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED">
-            <fa-icon class="pull-right" [icon]="getIconClass('author')" (click)="clickSortColumn('author')"></fa-icon>
-            <label placement="top" matTooltip="The editing author as collected by Dockstore">Author</label>
-          </th>
-          <th>
-            <fa-icon class="pull-right" [icon]="getIconClass('last_modified')" (click)="clickSortColumn('last_modified')"></fa-icon>
-            <label placement="top" matTooltip="Date of last update to Git reference">Date Modified</label>
-          </th>
+    <ng-container matColumnDef="valid">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="A version is valid if it has at least one valid descriptor file."
```

Comment:
This wasn't changed in this PR, but is this true anymore? 
@agduncan94  should this actually be "A version is valid if it has no validation messages"?

Reply:
I think this sentence makes more sense for tools, since a workflow would only have one descriptor. I think this meant if a tool version had a valid CWL but invalid WDL, it would be considered valid.

Your suggestion is definitely better. Maybe something like
"A version is valid if the descriptor file(s) have been successfully validated."